docs/gds-supported-platform/getting-started-gsp-local.md

```diff
@@ -1,54 +1,125 @@
 
-These instructions tell you how to set up a local GDS Supported Platform (GSP) cluster and host an app on that cluster.
+These instructions tell you how to set up a local instance of the GDS Supported Platform (GSP) and run the example `govuk-prototype-kit` app on that instance. You can use this process to test your app before deploying that app to a production environment. You will:
 
-This process is resource-intensive and you must set the resource amount used by Docker to the following:
-- 4 CPUs
-- 8 gigabytes of memory
+- [install prerequisite software](#install-prerequisite-software)
+- [create and run a local GSP instance](#create-and-run-a-local-gsp-instance)
+- [create a Dockerised `govuk-prototype-kit` app](#create-a-dockerised-govuk-prototype-kit-app)
+- [create a Helm chart](#create-a-helm-chart)
+- [deploy the app on to the local GSP instance](#deploy-the-app-on-to-the-local-GSP-instance)
+- [connect to the app](#connect-to-the-app)
+- [destroy the local instance](#destroy-the-local-instance)
 
-## Build a local GSP cluster
+This process should take no more than 2 hours. Contact the GSP team using the [#re-gsp Slack channel](https://gds.slack.com/messages/CDA7YSP0D) if it takes longer.
 
-1. Install [Homebrew](https://brew.sh/)
+<%= warning_text('Whilst setting up or running a local GSP instance is running, do not:<br>- power down your laptop<br>- put your laptop in standby mode<br>- connect or disconnect the Cisco VPN client') %>
 
-1. Clone the GSP repo:
+## Install prerequisite software
+
+1. Install [Homebrew](https://brew.sh/) and run the following to make sure you have the latest version of Homebrew.
+
+    ```
+    brew update
+    ```
+
+1. Go to the local repository folder and run the following to install the prerequisite software:
+
+    ```
+    brew bundle
+    ```
+
+1. Run the following to grant `driver superuser` privileges to the hypervisor:
+
+    ```
+    sudo chown root:wheel /usr/local/bin/docker-machine-driver-hyperkit && sudo chmod u+s /usr/local/bin/docker-machine-driver-hyperkit
+    ```
+
+1. Install [Docker](https://docs.docker.com) and configure Docker to have the following settings:
+
+    - CPU = 4
+    - Memory = 8 Gb
+    - Swap = 2 Gb
+
+1. Clone the GSP repository:
 
     ```
     git clone https://github.com/alphagov/gsp.git
     ```
 
-1. Install dependencies from the Brewfile at the root of the repo:
+1. If you already have either a `~/.minkube/machine/minikube` or `~/.minkube/machine/gocd` directory on your local machine,  run the following to ensure that Minikube is not running an existing cluster:
 
     ```
-    brew bundle --verbose
+    minikube stop -p CLUSTER
     ```
+    where `CLUSTER` can be either `minikube` or `gocd`.
 
-    After installing `docker-machine-driver-hyperkit` follow instructions on how to grant driver superuser privileges to the hypervisor.
+1. If the `kubeconfig` environment variable is assigned, run the following to unset this environment variable:
```

Comment:
KUBECONFIG should be uppercase as environment variable are case sensitive

Reply:
ok